e2e-tests/src/test/fee.rs

```diff
@@ -1,12 +1,8 @@
 use aleph_client::{
-    balances_transfer, get_next_fee_multiplier, get_tx_fee_info, send_xt, AccountId, AnyConnection,
-    AnyConnectionExt, Extrinsic, FeeInfo, RootConnection, SignedConnection, TransferTransaction,
-    XtStatus,
+    balances_transfer, get_next_fee_multiplier, AccountId, BalanceTransfer, CallSystem, FeeInfo,
+    GetTxInfo, ReadStorage, RootConnection, XtStatus,
 };
-use codec::Encode;
-use sp_core::Pair;
 use sp_runtime::{traits::One, FixedPointNumber, FixedU128};
-use substrate_api_client::{compose_call, compose_extrinsic, ExtrinsicParams, GenericAddress};
```

Comment:
An alternative would be that we'll extract methods that use any of those (so we can still get rid of `substrate-api-client` dep on `e2e-client`), as we do now, and don't introduce additional traits interfaces.